scripts/node-16kb.py

```
@@ -0,0 +1,44 @@
+NODE_ZIP_PATH = 'node-mobile.zip'
+
+def download_large_file(url, output_file):
+    with requests.get(url, stream=True) as response:
+        response.raise_for_status()  # Check for HTTP errors
+        with open(output_file, 'wb') as file:
+            for chunk in response.iter_content(chunk_size=8192):  # 8 KB chunks
+                if chunk:  # Filter out keep-alive chunks
+                    file.write(chunk)
+
+def calculate_md5(file_path):
+    md5_hash = hashlib.md5()
```

Ruff (0.14.0)

Probable use of insecure hash functions in `hashlib`: `md5`

(S324)